management/src/clusterm/manager/api.go

```diff
@@ -141,36 +137,35 @@ func validateAndSanitizeEmptyExtraVars(errorPrefix, extraVars string) (string, e
 	return extraVars, nil
 }
 
-func (m *Manager) nodesCommission(tags []string, sanitizedExtraVars string) error {
-	me := newWaitableEvent(newCommissionEvent(m, tags, sanitizedExtraVars))
+func (m *Manager) nodesCommission(req *APIRequest, extraVars string) error {
+	if !IsValidHostGroup(req.HostGroup) {
```

Comment:
there is a validate function `commonEventValidate()` before that. We can wrap it into commsionEvent specific validate function and do this validation in addition to common validation.